public/app/features/dashboard/components/PanelEditor/getVisualizationOptions.tsx

```diff
@@ -144,6 +152,19 @@ export function getVisualizationOptions(props: OptionPaneRenderProps): OptionsPa
   return Object.values(categoryIndex);
 }
 
+// create dynamic props
+function createDynamicProps(propsInfo: Record<string, string>, collection: Record<string, any>, sourceProps: any) {
+  let sourcePropsValue = lodashGet(sourceProps, propsInfo.path);
+  if (!isNil(sourcePropsValue) && propsInfo?.dynamic) {
```

Comment:
Why `propsInfo?` Seems like a typo. This function should never be called if bound is not defined on  field settings.